gallery/src/data-visualization/examples/ReactVisLineGraphExample.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useEffect, useState } from 'react';
+import { NxButton, NxLoadingSpinner, selectableColors } from '@sonatype/react-shared-components';
+import { XYPlot, VerticalGridLines, HorizontalGridLines, XAxis, YAxis, LineSeries,
+  Hint, LineSeriesPoint } from 'react-vis';
+import { randomNumberGenerator } from '../../util/jsUtil';
+import '../ReactVis.scss';
+
+export default function ReactVisLineGraphExample() {
+  const data: LineSeriesPoint[] = [
+    {x: 0, y: 8},
+    {x: 1, y: 5},
+    {x: 2, y: 4},
+    {x: 3, y: 9},
+    {x: 4, y: 1},
+    {x: 5, y: 7},
+    {x: 6, y: 6},
+    {x: 7, y: 3},
+    {x: 8, y: 2},
+    {x: 9, y: 0}
+  ];
+
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
I think this example is more complicated than it needs to be.  I'd remove the loading and the button that generates new data.

Reply:
> I think this example is more complicated than it needs to be. I'd remove the loading and the button that generates new data.

All examples are now simple with a static data set. The loading and update data buttons have been removed.
b2164bcc3751943a5e2340545f7c1c3c1a3a2133